### PR TITLE
build obr locally with a script

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

- [x] `build-locally.sh` script added.
- [x] `galasabld` tool is used if it's on the path, or built locally.

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: build-locally.sh
Comment (Jimbo4794) [inline build-locally.sh:155]: Remove gradle deps

**Review by Jimbo4794 — CHANGES_REQUESTED** [review]

File: build-locally.sh
Comment (Jimbo4794) [inline build-locally.sh:219]: Did we want a message to appear if cannot find on path or built? Just to let user know it cant find a required tool

**Comment by galasa-team** [comment]
Build successful

**Review by Jimbo4794 — CHANGES_REQUESTED** [review]
Looks good and will approve if the answer to the Q is no

File: build-locally.sh
Comment (techcobweb) [reply build-locally.sh:219]: already have:
```
info "The galasa build tool 'galasabld' or '$GALASA_BUILD_TOOL_NAME' is not on the path."
```

Comment (Jimbo4794) [reply build-locally.sh:220]: I am blind, approving

**Review by Jimbo4794 — APPROVED** [review]

**Comment by Jimbo4794** [comment]
@techcobweb approved and ready once built for you to merge

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**
